data-plane/receiver/src/test/java/dev/knative/eventing/kafka/broker/receiver/MockReactiveProducerFactory.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.knative.eventing.kafka.broker.receiver;
+
+import java.util.Properties;
+
+import org.apache.kafka.clients.producer.Producer;
+
+import io.cloudevents.CloudEvent;
+import io.vertx.core.Vertx;
+
+public class MockReactiveProducerFactory implements ReactiveProducerFactory<String, CloudEvent> {
+
+    @Override
+    public ReactiveKafkaProducer<String, CloudEvent> create(Vertx v, Properties config) {
+        return new MockReactiveKafkaProducer<String, CloudEvent>(config);
+    }
+
+    @Override
+    public ReactiveKafkaProducer<String, CloudEvent> create(Vertx v, Producer<String, CloudEvent> producer) {
+        return new MockReactiveKafkaProducer<String, CloudEvent>(producer);
+    }
+
+    public static ReactiveKafkaProducer<String, CloudEvent> createStatic(Vertx v, Properties config) {
+        return new MockReactiveKafkaProducer<String, CloudEvent>(config);
+    }
+
+    public static ReactiveKafkaProducer<String, CloudEvent> createStatic(Vertx v,
+            Producer<String, CloudEvent> producer) {
+        return new MockReactiveKafkaProducer<String, CloudEvent>(producer);
+    }
```

Comment:
Do we need the static methods or we can use the instance methods? with like:

```
new MockReactiveProducerFactory().create(...)
```

Reply:
I have removed the `staticCreate` and use `new MockReactiveProducerFactory().create(...)` instead.